scripts/umap_reducer.py

```diff
@@ -198,16 +229,19 @@ def main(fmat, fxyz, ftags, fcolor, colorscol, prefix, output, peratom, keepraw,
     parser.add_argument('--dim1', type=int, default=0, help='Plot the projection along which principle axes')
     parser.add_argument('--dim2', type=int, default=1, help='Plot the projection along which principle axes')
     parser.add_argument('--projectatomic', type=str2bool, nargs='?', const=True, default=False,
-                        help='Building the UMAP projection based on atomic descriptors instead of global ones (True/False)')
+                        help='Building the UMAP projection based on atomic descriptors instead of global ones '
+                             '(True/False)')
     parser.add_argument('--plotatomic', type=str2bool, nargs='?', const=True, default=False,
                         help='Plot the manifold coordinates of all atomic environments (True/False)')
     parser.add_argument('--adjusttext', type=str2bool, nargs='?', const=True, default=False,
                         help='Do you want to adjust the texts (True/False)?')
+    parser.add_argument('--use_morgan', type=bool, default=False, help='Whether to use Morgan fingerprints')
```

Comment:
Extra flag for Morgan fingerprints, defaults to False.

Reply:
This is quite awkward. In the redesign of the code, we have Global_Descriptors objects, and you can declare new ones. I think Morgan fingerprints fits perfectly there.

Reply:
Great stuff!